AAUnlimited/MemMods/AAPlay/Events/HInjections.cpp

```diff
@@ -67,6 +74,34 @@ bool __stdcall TickRedirect(ExtClass::HInfo* hInfo) {
 			loc_activeFaceXX = active->m_xxFace;
 			loc_passiveFaceXX = passive->m_xxFace;
 		}
+		// Detecting cumshot pose stage
+		cumshotStart = (byte*)ExtVars::ApplyRule(cumshotStartOffset);
+		if (!cumshotStarted && *cumshotStart == 1) {
+			LUA_EVENT_NORET("start_h_cumshot", hInfo);
+			cumshotStarted = true;
+			DrawD3D::canRender = false; // Fix against drawing on naked skin in cumshot stage
+			renderDelayStarted = false;
+		}
+		if (cumshotStarted) {
+			cumshotSprays = (byte*)ExtVars::ApplyRule(cumshotEndOffset); // Current count of released sprays
+			if (*cumshotSprays == 0)
+				loc_cumshotSprays = 0;
+			else if (*cumshotSprays != loc_cumshotSprays) {
+				loc_cumshotSprays = *cumshotSprays;
+				LUA_EVENT_NORET("cumshot_every_spray", loc_cumshotSprays);
```

Comment:
Are you going to use this for something?

Reply:
Currently not used. I thought before that someone might need this feature. However, now I also think that it is unlikely that anyone will need it. I'll remove/comment this line with event, ok?